test/e2e/recordPlaywright.js

```diff
@@ -0,0 +1,64 @@
+const fs = require("fs");
+
+const playwright = require("@recordreplay/playwright");
+const cli = require("@recordreplay/recordings-cli");
+const _ = require("lodash");
+const { waitUntil } = require("./waitForTime");
+
+const verbose = true;
+
+async function record(browserName, example) {
+  if (verbose) {
+    console.log(`Recording Example:`, example, browserName);
+  }
+
+  const browser = await playwright[browserName].launch({
+    headless: false,
+  });
+
+  const context = await browser.newContext();
+  const page = await context.newPage();
+  await page.goto(example);
+  await waitUntil(() => page.evaluate(() => window.Example__Finished));
```

Comment:
Does this put a 10 second timer on all tests by default? Usually `waitUntil` only waits for 10 seconds I think.

Reply:
this is just for the examples, but might be something to bump